$NetBSD$

Log to syslog, instead of (perhaps) not logging.

Set a pid_file for use by rc.d/mosquitto.

--- mosquitto.conf.orig	2019-06-18 11:45:59.000000000 +0000
+++ mosquitto.conf
@@ -159,6 +159,7 @@
 # being run automatically on boot with an init script and
 # start-stop-daemon or similar.
 #pid_file
+pid_file @VAR_RUN@/mosquitto/mosquitto.pid
 
 # Set to true to queue messages with QoS 0 when a persistent client is
 # disconnected. These messages are included in the limit imposed by
@@ -586,6 +587,7 @@
 # "log_dest none" and neither stdout nor stderr logging is available.
 # Use "log_dest none" if you wish to disable logging.
 #log_dest stderr
+log_dest syslog
 
 # Types of messages to log. Use multiple log_type lines for logging
 # multiple types of messages.
